scripts/trace_runfolder.py

```diff
@@ -3,6 +3,7 @@
 import requests
 import json
 import argparse
+import datetime
```

Comment:
I'd suggest using `from datetime import datetime` in order to not have to do the double `datetime.datetime` below.